manifests/agent/postfix_ipv4.pp

```diff
@@ -1,4 +1,7 @@
 class classroom::agent::postfix_ipv4 {
+  package {'postfix':
```

Comment:
@binford2k postfix was installed by the Puppetfactory Dockerfile, but we only need it in the classroom, so I moved it here. It needs to be pre installed or the augeas resource fails.